src/app/item-page/orcid-page/orcid-sync/orcid-setting.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+import { Operation } from 'fast-json-patch';
+import { switchMap } from 'rxjs/operators';
+import { AuthService } from '../../../core/auth/auth.service';
+import { ItemDataService } from '../../../core/data/item-data.service';
+import { RemoteData } from '../../../core/data/remote-data';
+import { ResearcherProfileService } from '../../../core/profile/researcher-profile.service';
+import { Item } from '../../../core/shared/item.model';
+import { getFinishedRemoteData, getFirstCompletedRemoteData } from '../../../core/shared/operators';
+import { NotificationsService } from '../../../shared/notifications/notifications.service';
+
+@Component({
+  selector: 'ds-orcid-setting',
+  templateUrl: './orcid-setting.component.html',
+  styleUrls: ['./orcid-setting.component.scss']
+})
+export class OrcidSettingComponent implements OnInit {
+
+  messagePrefix = 'person.page.orcid';
+
+  currentSyncMode: string;
+
+  currentSyncPublications: string;
+
+  currentSyncFundings: string;
+
+  syncModes: { value: string, label: string }[];
+
+  syncPublicationOptions: { value: string, label: string }[];
+
+  syncFundingOptions: {value: string, label: string}[];
+
+  syncProfileOptions: { value: string, label: string, checked: boolean }[];
+
+  item: Item;
+
+  constructor(private researcherProfileService: ResearcherProfileService,
+    protected translateService: TranslateService,
+    private notificationsService: NotificationsService,
+    public authService: AuthService,
+    private route: ActivatedRoute,
+    private itemService: ItemDataService
+  ) {
+     this.itemService.findById(this.route.snapshot.paramMap.get('id'), true, true).pipe(getFirstCompletedRemoteData()).subscribe((data: RemoteData<Item>) => {
+      this.item = data.payload;
+     });
```

Comment:
This component is part of the ItemPageRouting module. The item is already part of the route data. You can get the item from the route [like this](https://github.com/4Science/dspace-angular/blob/804930fbe2b5cca46343d24bf7486034cf5cc4f8/src/app/item-page/simple/item-page.component.ts#L80-L83).